compiler/dfa.nim

```diff
@@ -639,6 +640,8 @@ proc gen(c: var Con; n: PNode) =
         genMagic(c, n, s.magic)
       else:
         genCall(c, n)
+      if sfNoReturn in n.sons[0].sym.flags:
```

Comment:
sfNoReturn is better indicator for noReturn then `mExit`

Reply:
Ok, but in the longer run, if we depend on 'sfNoReturn' we should also check that ``.noReturn`` procs do not lie about this attribute.